packit/upstream.py

```diff
@@ -412,14 +397,7 @@ def _create_archive_using_default_way(self, dir_name, env, version) -> str:
 
         :return: name of the archive
         """
-        archive_extension = self.get_archive_extension(dir_name, version)
-        if archive_extension not in COMMON_ARCHIVE_EXTENSIONS:
-            raise PackitException(
-                "The target archive doesn't use a common extension ({}), "
-                "git archive can't be used. Please provide your own script "
-                "for archive creation.".format(", ".join(COMMON_ARCHIVE_EXTENSIONS))
-            )
-        archive_name = f"{dir_name}{archive_extension}"
+        archive_name = f"{dir_name}{DEFAULT_ARCHIVE_EXT}"
```

Comment:
How sure are we that the default extension will work for everyone?

And another side note: I don't think a command bellow will actually do the `.gz` part. IIRC, by default it's only `tar`.

Reply:
Apologies, the last remark was wrong, `git archive` will infer from the output name the format, and use that, and `tar.gz` is something that `git` understands.

Reply:
> How sure are we that the default extension will work for everyone?

It definitely won't work for everyone, this is too much black magic. That's why people can generate the tarball on their own and not use this code.

> ~And another side note: I don't think a command bellow will actually do the `.gz` part. IIRC, by default it's only `tar`.~

yeah, tests say something else :P